db/migrate/20131118232818_encrypt_miq_database_registration_http_proxy_password_field.rb

```diff
@@ -1,5 +1,7 @@
 class EncryptMiqDatabaseRegistrationHttpProxyPasswordField < ActiveRecord::Migration
-  class MiqDatabase < ActiveRecord::Base; end
+  class MiqDatabase < ActiveRecord::Base
+    self.inheritance_column = :_type_disabled # disable STI
```

Comment:
I maintain my firm belief that this isn't necessary; if the type column's not there now, it never will be for this migration.